### PR TITLE
search page code

### PR DESCRIPTION
**Pull request by JuniorResource**
Hello Gourav

I have raised PR for search page

### CODE REVIEW

File: src/searchPage.java [hunk 1]
```diff
@@ -5,6 +5,9 @@ public static void main(String[] args) {
 		
 		System.out.println("We are in search page");
 		System.out.println("Bye");
+
+		int a=12;
```
Comment (GouravPersonalGit) [inline src/searchPage.java:9]: Why using this  a variable

**Review by GouravPersonalGit — CHANGES_REQUESTED** [review]
Hello Junior,

Please make changes that mentioned in review comments

File: src/searchPage.java (hunk 1 above)
Comment (JuniorResource) [reply src/searchPage.java:8]: added the implementation of a variable

**Comment by GouravPersonalGit** [comment]
Search page merge to master

**Pull request merged**